Updating CFT spec with renaming and `CFTokenAuthorize` tx (#137)
XRPLF/XRPL-Standards: XLS-0033d-compact-fungible-tokens/README.md

| Flag Name         | Flag Value | Description                                                             |
|-------------------|------------|-------------------------------------------------------------------------|
| `lsfLocked`       | `0x0001`   | If set, indicates that the CFT owned by this account is currently locked and cannot be used in any XRP transactions other than sending value back to the issuer. When this flag is set, the `LockedAmount` must equal the `CFTAmount` value. |

> | `lsfAuthorized`       | `0x0002`   | (Only applicable for allow-listing) If set, indicates that the issuer has authorized the holder for the CFT, this is an flag that will be set through a `CFTokenAuthorize` transaction. This flag can also be unset through a `CFTokenAuthorize` transaction by setting the `tfUnauthorize` flag.|

Reviewer: ```suggestion
| `lsfAuthorized`       | `0x0002`   | (Only applicable for allow-listing) If set, indicates that the issuer has authorized the holder for the CFT. This flag can be set using a `CFTokenAuthorize` transaction; it can also be "un-set" using a `CFTokenAuthorize` transaction specifying the `tfUnauthorize` flag. |
```

Reviewer: There was a minor typo in the original, but I decided to suggest a few more edits.

Indicates the new transaction type **`CFTokenIssuanceCreate`**. The integer value is `25 (TODO)`.

| Field Name         | Required?    | JSON Type | Internal Type |
| ------------------ | ------------ | --------- |---------------|
| `AssetScale`       | ️           | `number`  | `UINT8`       |

> An asset scale is the difference, in orders of magnitude, between a standard unit and a corresponding fractional unit. More formally, the asset scale is a non-negative integer (0, 1, 2, …) such that one standard unit equals 10^(-scale) of a corresponding fractional unit. If the fractional unit equals the standard unit, then the asset scale is 0.

Reviewer: ```suggestion
An asset scale is the difference, in orders of magnitude, between a standard unit and a corresponding fractional unit. More formally, the asset scale is a non-negative integer (0, 1, 2, …) such that one standard unit equals 10^(-scale) of a corresponding fractional unit. If the fractional unit equals the standard unit, then the asset scale is 0. Note that this value is optional, and will default to `0` if not supplied.
```
